pageserver/src/tenant/timeline.rs

```diff
@@ -2878,6 +2870,14 @@ impl Timeline {
             ancestor_gc_info.insert_child(timeline_id, metadata.ancestor_lsn(), is_offloaded);
         }
 
+        let relsize_snapshot_cache_capacity = {
+            let loaded_tenant_conf = tenant_conf.load();
+            loaded_tenant_conf
+                .tenant_conf
+                .relsize_snapshot_cache_capacity
+                .unwrap_or(conf.default_tenant_conf.relsize_snapshot_cache_capacity)
+        };
```

Comment:
Please plug into `Timeline::tenant_conf_updated` and resize the cache when the config changes.
Otherwise, the config change still requires a restart.

Reply:
Done